dbf2csv/controllers/controllers.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
Inconsistent copyright header - in other files you have used:
```
# Copyright 2017 INGETIVE (<https://ingetive.com>)
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Reply:
Ok, thanks

Reply:
Ok, thanks